app/utils/queries_zaken_api.py

```diff
@@ -52,7 +52,9 @@ def get_fraudprediction_cases_from_AZA_by_model_name(model_name):
             queryParams = {
                 "open_cases": "true",
                 "state_types": [str(state.get("id", 0)) for state in state_types],
-                "theme": team_settings.get("zaken_team_id", None),
+                "theme": team_settings.get("zaken_team_id")
+                if team_settings is not None
+                else None,
```

Comment:
Even checken of ik deze code goed lees:
1. haal de team-settings op op basis van het `zaken_team_id`
2a. als de `team_settings` niet `None` zijn: _lege `if()`-clause
2b. als de `team_settings` wel `None` zijn, kies als standaard lengte van de lijst 1000 entries

Wil je die wel/niet `None` check niet _voor_ de `.get()` doen?

Reply:
Oh.... wacht. Kan Python net als Perl "conditionals at the end of a line"?

`print "plop" unless $plop;` dat?

Reply:
Dit is Pythonesque: `"theme": team_settings.get("zaken_team_id") if team_settings is not None else None,` maar de linter zet het op meerdere regels.
`get()` haalt een waarde uit een dictionary.

Reply:
okidoki :)